wp-content/themes/missilethreat/assets/_scss/abstracts/_placeholders.scss

```
@@ -52,6 +52,7 @@
 %font-ui-heading-md {
   @include font-clamp(20px, 24px);
   line-height: 1.14;
+  font-weight: 400;
```

```suggestion
  font-weight: normal;
```

It's more common to use `normal` instead of `400` (normal defaults to 400).